test/tools/ossfuzz/CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ if (OSSFUZZ)
             /usr/include/libprotobuf-mutator
     )
     target_link_libraries(abiv2_proto_ossfuzz PRIVATE solidity
-            evmone intx ethash evmc-instructions
+            evmc evmone intx ethash evmc-instructions
```

Comment:
I'm not sure if adding `evmc` is required because the CI and oss-fuzz docker images contain a system-wide installation of `evmc` at `/usr`. Looking at the present failing build of ossfuzz, I get a feeling the linking error is because of a boost dependency that was newly introduced with this PR, but I'm not sure.

Reply:
The error was that it could not find evmc_load

Reply:
Which boost dependency are you talking about? Maybe it was a rebase problem.

Reply:
> The error was that it could not find evmc_load

Ah, I see. I overlooked this change that adds `evmc` as an dependency for the fuzzing target. Thank you for the clarification.

https://github.com/ethereum/solidity/pull/7010/files#diff-e4cd21bbaea28071e286e4adfd6ec79aR40

> Which boost dependency are you talking about? Maybe it was a rebase problem.

I would hazard the guess that it is boost unit test library?

```
Linking CXX executable abiv2_proto_ossfuzz
/usr/include/boost/test/unit_test_log.hpp:224: error: undefined reference to 'vtable for boost::unit_test::unit_test_log_t'
/usr/bin/ld.gold: the vtable symbol may be undefined because the class is missing its key function
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [test/tools/ossfuzz/CMakeFiles/abiv2_proto_ossfuzz.dir/build.make:158: test/tools/ossfuzz/abiv2_proto_ossfuzz] Error 1
make[2]: *** [CMakeFiles/Makefile2:866: test/tools/ossfuzz/CMakeFiles/abiv2_proto_ossfuzz.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:910: test/tools/ossfuzz/CMakeFiles/ossfuzz_abiv2.dir/rule] Error 2
make: *** [Makefile:359: ossfuzz_abiv2] Error 2
Exited with code 2
```